infra/resources/_modules/web_apps/etl_func.tf

```diff
@@ -23,7 +23,10 @@ module "etl_func" {
     EVENTHUB_CONNECTION_URI        = var.app_settings.eventhub_connection_uri,
     MESSAGE_CONTENT_CONTAINER_NAME = "message-content",
     MESSAGE_EVENTHUB_NAME          = "io-p-itn-com-etl-messages-evh-01"
-    PDV_TOKENIZER_API_KEY = "@Microsoft.KeyVault(VaultName=${var.common_key_vault.name};SecretName=func-elt-PDV-TOKENIZER-API-KEY)"
+    PDV_TOKENIZER_API_KEY          = "@Microsoft.KeyVault(VaultName=${var.common_key_vault.name};SecretName=func-elt-PDV-TOKENIZER-API-KEY)"
+    REDIS_URL                      = var.redis_cache.url
+    REDIS_PASSWORD                 = var.redis_cache.access_key
```

Comment:
Probably this must be REDIS_ACCESS_KEY ?

Reply:
yes you are right